e2e/docker/compose.yaml

```diff
@@ -0,0 +1,45 @@
+name: flwr-containers-e2e-test
+
+services:
+  # create a SuperLink service
+  superlink:
+    image: docker.io/flwr/superlink:1.9.0.dev20240513
+    ports:
+      - "9091:9091"
+      - "9092:9092"
+    networks:
+      - flwr-net
+    hostname: flwr-superlink
```

Comment:
When using the Docker-Compose the default network by default uses the service name (`superlink`) as hostname. Therefore we can remove the `networks` config in each service and change the SuperLink URL to `"superlink:9092”` or `"superlink:9091”`. 
```suggestion
```

Reply:
Ah, cool! Added.